__test__/index.test.js

```diff
@@ -0,0 +1,18 @@
+
+// const  isEven  = require("../index"); 
+
+/*
+*  importing legacy function giving RangeError: Maximum call stack size exceeded
+*  so i'm gonna use tiny part of function to write test cases, 
+*/
+
+const isEven = require("./index.tiny");
+
+describe('check if a number is even or odd', () => {
+    it("1 should be false", ()=>{
+        expect(isEven(1)).toBeFalsy();
+    }) 
+    it("2 should be true", ()=>{
+        expect(isEven(1)).toBeFalsy();
```

Comment:
this test doesn't even do what you say it does, are you even trying?

Reply:
lol i was watching Samuel miller videos when i was doing this,just pushed a fix.